xwiki-platform-core/xwiki-platform-vfs/xwiki-platform-vfs-ui/src/main/resources/Macros/VFSTreeMacro.xml

```diff
@@ -233,18 +338,30 @@
         <unmodifiable>0</unmodifiable>
         <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
       </name>
+      <type>
+        <disabled>0</disabled>
+        <name>type</name>
+        <number>5</number>
+        <prettyName>Parameter type</prettyName>
+        <size>60</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </type>
     </class>
     <property>
       <defaultValue/>
     </property>
     <property>
-      <description>The reference to the file in an attachment archive (e.g. "attach:space.page@attachment/my/file.txt")</description>
+      <description>The reference to the file in an attachment archive (e.g. "[[attachment/my/file.txt")&gt;&gt;attach:space.page@attachment/my/file.txt")]]</description>
```

Comment:
Was that change on purpose ?

Reply:
For each xar file change, I edited a running wiki to copy the value of `defaultCategory` to `defaultCategories` and remove the deprecated field and did a xar export. So the change is the result of this process. But indeed it looks strange, I'll check what happened!

Reply:
Indeed I did another export and the description stays the same, reverting this change.